keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -483,6 +508,9 @@ For each of them, fill in the following information by copying the below templat
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+- Check `plugin_execution_duration_seconds{plugin="PodTopologySpread"}` to see if latency increased. 
+- Check `schedule_attempts_total{result="error|unschedulable"}` to see if the number of attempts increased.
```

Comment:
What should I do if I see problems in either of the metrics?

Reply:
Updated.